src/index.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference types="webpack-env" />
+
+import type { Meta, StoryContext as _SB_StoryContext, StoryObj } from '@storybook/svelte';
+
+import Story from './components/Story.svelte';
+
+export type { Story };
+export { setTemplate } from './components/context.svelte.js';
+
+// FIXME: We don't use webpack anymore(?)
```

Comment:
Another decision I need help with.

Should we preserve support for `webpack`?
I don't have an example reproduction, where I can test neccessary changes after refactor to addon's loader for the webpack.

For the moment I've put it on hold.